runtime/dancebox/tests/integration_test.rs

```diff
@@ -1983,3 +1986,34 @@ fn check_well_known_keys() {
         frame_support::storage::storage_prefix(b"Session", b"CurrentIndex")
     );
 }
+
+#[test]
+fn test_invulnerables_migration() {
+    ExtBuilder::default()
+        .with_balances(vec![
+            // Alice gets 10k extra tokens for her mapping deposit
+            (AccountId::from(ALICE), 210_000 * UNIT),
+            (AccountId::from(BOB), 100_000 * UNIT),
+        ])
+        .with_collators(vec![
+            (AccountId::from(ALICE), 210 * UNIT),
+            (AccountId::from(BOB), 100 * UNIT),
+        ])
+        .with_config(pallet_configuration::HostConfiguration {
+            max_collators: 100,
+            min_orchestrator_collators: 2,
+            max_orchestrator_collators: 2,
+            collators_per_container: 2,
+        })
+        .build()
+        .execute_with(|| {
+            let invulnerables_collator_selection = CollatorSelection::invulnerables();
```

Comment:
One suggestion now that we got this test working: remove the

```
.with_collators(vec![
            (AccountId::from(ALICE), 210 * UNIT),
            (AccountId::from(BOB), 100 * UNIT),
        ])
```

and use `put_storage_value` like in https://github.com/moonbeam-foundation/moonbeam/blob/446faeb04153d127211a499244c900a1a5c71e54/pallets/asset-manager/src/tests.rs#L468. That way we are ready to remove the CollatorSelection in the following PR without having to change the test

Reply:
done